app/models/flood_risk_engine/organisation.rb

```diff
@@ -2,6 +2,7 @@ module FloodRiskEngine
   class Organisation < ActiveRecord::Base
     belongs_to :contact
     has_one :enrollment, dependent: :restrict_with_exception
+    has_many :partners
```

Comment:
Use a `where` clause on the `has_many` call doesn't work, because the `where` is applied to the class being joined to, not the model hosting the `has_many` declaration.

STI would make sense if the variation between the behaviours of different organisation types was greater than this one instance. Adding 6 class object to handle one small variant is overkill.